easypermissions/src/test/java/pub/devrel/easypermissions/TestActivity.java

```diff
@@ -0,0 +1,6 @@
+package pub.devrel.easypermissions;
+
+import android.support.v7.app.AppCompatActivity;
+
+class TestActivity extends AppCompatActivity {
```

Comment:
Could you please add one-line JavaDoc comments to this class and to `TestApplication` to specify why we need them?  When reading this PR it's obvious, but it may not be obvious to future code readers.